### DISCUSSION TITLE
Searching in multiple folders at once

### DISCUSSION THREAD

**Discussion by mathieucloutier86** [dnGrep/dnGrep#1194] [Q&A] [answered]
Hello,

Perhaps I'm missing something, but I was wondering if it was possible to search through a selection of folders during a single search session.  If so, is it possible to copy paste a list of directory path. This would be very convenient to search through project notes that are not nested under a same parent folder.

Best regards,
Mathieu 

PS:  I wish to apologize for my lack of response in the last few months. A lot of positive change in my life (purchase of a property and birth of my firstborn) kept my away longer than I thought. Rest assured that my interest toward the evolution of dngrep has not diminished in the slightest.

**Comment by fade2gray** [comment, marked as answer]
Separate multiple paths with commas or semicolons.  https://github.com/dnGrep/dnGrep/issues/976#issuecomment-1791881103

![multiple_paths](https://github.com/user-attachments/assets/2e972f7b-dda6-4fcb-916d-1512bed7f575)

**Reply by mathieucloutier86** [reply]
Awesome ! Thanks a lot !
